Containerfile

```diff
@@ -23,18 +21,16 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
     LANG=en_US.UTF-8 \
     PIP_NO_CACHE_DIR=off
 
-WORKDIR ${APP_ROOT}
+WORKDIR /app-root
 
 COPY --from=lightspeed-rag-content /rag/vector_db/ocp_product_docs ./vector_db/ocp_product_docs
 COPY --from=lightspeed-rag-content /rag/embeddings_model ./embeddings_model
 
 # Add explicit files and directories
 # (avoid accidental inclusion of local directories or env files or credentials)
-COPY pyproject.toml pdm.lock runner.py ./
-RUN pip3.11 install --no-cache-dir --upgrade pip pdm==2.18.1 \
-    && pdm config python.use_venv false \
-    && pdm sync --global --prod -p ${APP_ROOT}
+COPY pyproject.toml pdm.lock runner.py requirements.txt ./
```

Comment:
Do we still need to copy pyproject.toml and pdm.lock?

Reply:
no, lets remove them